tests/integration/tier1/destructive/kernel-excluded-in-conf/main.fmf

```diff
@@ -0,0 +1,13 @@
+summary: |
+    Exclude kernel, kernel-core and redhat-release-server packages in yum conf
+
+description: |
+  This tests verifies that it is possible to do a full conversion when
+  the kernel packages and redhat-release-server is set to be excluded
+  in the yum configuration file.
+
+tag+:
+    - kernel
+
+/kernel_excluded_in_conf:
```

Comment:
Given we don't exclude just kernel, but also redhat-release-server, I'm thinking if this would make sense to be more generic?
e.g. `/exclude_packages_in_yum_conf:`

Reply:
Ok, I changed it to `/yum_conf_exclude_packages` as we already have a test that starts with "exclude_packages.."